README.md

```diff
@@ -263,9 +263,14 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // The client id given to the application by OIDC provider.
                 "client_id": "irods_http_api",
 
-                // The client secret used for accessing the introspection endpoint
+                // The client secret used for accessing the introspection endpoint.
+                // Optional unless running as a protected resource.
                 "client_secret": "xxxxxxxxxxxxxxx",
 
+                // The OIDC mode the HTTP API will run as.
+                // The value has to be "client" or "protected_resource"
```

Comment:
Missing period.

Reply:
Resolving in favor of https://github.com/irods/irods_client_http_api/pull/252#discussion_r1513413216